swagger.yaml

```diff
@@ -972,8 +1018,10 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
       security:
-        - oauth2:
+        - europace_oauth2_legacy:
             - API
+        - europace_oauth2:
+            - unterlagen:freigabe:lesen
```

Comment:
Das *Lesen* des Freigabestatus ist eigentlich nur für den Freigebenden (scope: unterlagen:unterlage:freigeben) relevant (so ähnlich wie der Kategorisierungsstatus bei "Hochladen"). Das *Setzen* der Freigabe sollte allerdings wohl schon im Scope "unterlage:freigabe:...) liegen. Keine Ahnung, ob man das hier so genau abbilden will -- und wie...

Reply:
in der UAkte zeigen wir den Status immer an, von daher gehe ich davon aus das der Status auch für Clients interessant ist, die nur die Informationen anzeigen wollen.